nevergrad/functions/ml/mlfunctionlib.py

```diff
@@ -15,6 +15,7 @@
 from nevergrad.parametrization import parameter as p
 from ..base import ExperimentFunction
 
+from tensorflow import keras
```

Comment:
This line can not work (tensorflow not in the native dependences).
Move the import below (see other comment).

Reply:
Thx I have updated the code -- I put the import in the paremetrization and in the init, which may not be optimal, but I didn't find a better way ..